### PR TITLE
Improve `__all__` handling code

### PR DESCRIPTION
**Pull request by hakancelik96**

### CODE REVIEW

File: unimport/scan.py [hunk 1]
```diff
@@ -112,20 +114,24 @@ def visit_Attribute(self, node):
...
+            if node.targets[0].id == "__all__" and isinstance(
+                node.value, (ast.List, ast.Tuple)
+            ):
                 for item in node.value.elts:
-                    if sys.version_info.minor == 8:
+                    get_value = False
+                    if (
+                        PY38_PLUS
+                        and isinstance(item, ast.Constant)
+                        and isinstance(item.value, ast.Str)
```
Comment (isidentical) [inline unimport/scan.py:126]: no

File: unimport/scan.py [hunk 2]
```diff
@@ -112,20 +114,24 @@ def visit_Attribute(self, node):
...
-            pass
-        else:
-            if is_all == "__all__":
+        if isinstance(node.targets[0], ast.Name):
+            if node.targets[0].id == "__all__" and isinstance(
+                node.value, (ast.List, ast.Tuple)
+            ):
                 for item in node.value.elts:
-                    if sys.version_info.minor == 8:
+                    get_value = False
```
Comment (isidentical) [inline unimport/scan.py:122]: get_vaue=None

File: unimport/scan.py [hunk 3]
```diff
@@ -112,20 +114,24 @@ def visit_Attribute(self, node):
...
+                        and isinstance(item.value, ast.Str)
+                    ):
                         get_value = item.value
-                    else:
+                    elif isinstance(item, ast.Str):
                         get_value = item.s
-                    self.names.append(
-                        {"lineno": node.lineno, "name": get_value}
-                    )
+                    if get_value:
```
Comment (isidentical) [inline unimport/scan.py:131]: else:continue

**Comment by hakancelik96** [comment]
I close this pr and I will send a new

**Pull request closed without merging**